blocks/footer/footer.css

```diff
@@ -1,14 +1,230 @@
+/* stylelint-disable no-descending-specificity */
 footer {
-  padding: 2rem;
-  background-color: var(--overlay-background-color);
-  font-size: var(--body-font-size-s);
+  background-color: #27292b;
+  align-items: flex-start;
+  flex-wrap: wrap;
+  width: 100%;
+  margin: 0 auto;
+  padding: 40px 0 100px;
```

Comment:
padding and margins, insofar as they are re-used across the site, should also be put into spacing CSS vars.